notebooks/demo2/time-anlysis.ipynb

```diff
@@ -0,0 +1,1187 @@
+{
```

Comment:
<p>is this in seconds</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/os-climate/aicoe-osc-demo/pull/210/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='20'/>

Reply:
<p>yes</p>